### ISSUE TITLE
infra: Some disks are not being recognized as RO/RW and containers are failing to get raised

### ISSUE THREAD

**Issue by gustavovalverde** [ZcashFoundation/zebra#5644]
## Motivation

We've been reducing the amount of errors happening in CI, and in GCP logs, to reduce the spectrum of possibilities causing SSH errors from CI.

One of the latest errors is:

```
Failed to start container: Volume pd-0: Could not determine if the GCE Persistent Disk zebrad-cache-7b73aa0 is attached read-only or read-write
```

### Designs

- Verify if there's documentation on how to properly mount disks on Compute Engine
- Validate if we're mounting disks in a standard way for all instances in CI and CD.

**Comment by mpguerra** [comment]
@gustavovalverde can we please get an estimate here?

**Comment by gustavovalverde** [comment]
This wasn't fixed by PR #5670, just the other ticket referenced by it was fixed.
